config/projects.yml

```diff
@@ -139,14 +139,9 @@ datagrid-bundle:
         symfony: ['2.8', '3.2', '3.3', '3.4']
 
 doctrine-extensions:
-  excluded_files:
-    - README.md
-  docs_target: false
   branches:
     master:
       php: ['7.1', '7.2']
-    1.x:
```

Comment:
Wow, what the hell is going on here? :-D

We can't just remove an already existing package and start an empty one.

Why do we have this: https://github.com/sonata-project/sonata-doctrine-extensions with `sonata-project/doctrine-extensions` as package name and now we have https://github.com/sonata-project/doctrine-extensions ? This makes no sense and will be very confusing.

Reply:
You're right. Think we can remove the new https://github.com/sonata-project/doctrine-extensions repo and rename the existing one (?).

Reply:
@core23 I can rename repositories, but not delete them.